pyproject.toml

```diff
@@ -6,8 +6,7 @@
 line-length = 120
 
 target-version = [
-    'py37', # used by Blender 2.80
-    'py39', # used by Blender 3.0.1
+    'py310', # used by Blender 3.1
```

Comment:
probably should be changed later, for now the repo is still for "old" blender?

Reply:
well the formatter needs to be 3.10 supported, plus this is going into the V5 branch, i figured we would merge the formatting changes into master until we were ready with V5